variables.tf

```diff
@@ -15,3 +15,12 @@ Encrypts all data at the object level as it is being written to S3, automaticall
 The way you access objects (presigned URLs, listing objects, and retrieving objects) all works the same way as if they were not encrypted.
 EOF
 }
+
+variable "public_read_only" {
```

Comment:
I think you will need to update `aws_s3_bucket_public_access_block` as well. I suspect that s3 uploads will fail.